vumi/transports/dmark/dmark_ussd.py

```diff
@@ -113,8 +113,20 @@ def handle_raw_inbound_message(self, request_id, request):
             log.msg('Unhappy incoming message: %r' % (errors,))
             self.finish_request(
                 request_id, json.dumps(errors), code=http.BAD_REQUEST)
+            yield self.add_status(
+                component='request',
+                status='down',
+                type='invalid_inbound_fields',
+                message='Invalid inbound fields',
+                details=errors)
             return
 
+        yield self.add_status(
+            component='request',
+            status='ok',
+            type='request_decoded',
+            message='Request decoded',)
```

Comment:
A little bikesheddy, but wondering if we should use 'parsed' instead of 'decoded', 'decoded' might imply character encoding or data deserializing which both seem like subsets of parsing.